build.sbt

```diff
@@ -379,4 +379,4 @@ lazy val sslConfig = project
   )
 
 addCommandAlias("check", "; scalafmtSbtCheck; scalafmtCheckAll; compile:scalafix --check; test:scalafix --check")
-addCommandAlias("fix", "; scalafmtSbt; scalafmtAll; compile:scalafix; test:scalafix; example/mdoc")
+addCommandAlias("fix", "; compile:scalafix ; test:scalafix; scalafmtSbt; scalafmtAll; example/mdoc")
```

Comment:
Is the space `compile:scalafix ;` important or is it just a typo?

Reply:
yeah, that's a typo introduced by me flipping the order
fixed :smile: